src/tests/Results/ResultsView.test.tsx

```diff
@@ -8,7 +8,7 @@ describe('Results View', () => {
   it('Should match snapshot', () => {
     render(<ResultsView />);
 
-    expect(screen).toMatchSnapshot();
+    expect(document.body).toMatchSnapshot();
```

Comment:
[question] is there a reason why we chnaged from screen to document.body

Reply:
when using screen, sometimes it shows an incorrect snapshot: 

```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results View Should match snapshot 1`] = `
Object {
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
...
```